dev_guide/projects.adoc

```diff
@@ -194,3 +194,160 @@ To delete a project using the CLI:
+xref:granting-project-access-using-the-web-console[web console].
+
+[[granting-project-access-using-the-web-console]]
+=== Granting Project Access Using the Web Console
+
+After xref:project-collaboration-adding-collaborators[adding the collaborator]
+to your {product-title} Pro subscription, you can grant project access to the
+collaborator using the web console.
+
+. From within a project, click *Resources*, then *Membership*.
+
+. Add roles (for example, *view*, *edit*, or *manage*) to the user you want to
+grant access to.
++
+See
+xref:../architecture/additional_concepts/authorization.adoc#architecture-additional-concepts-authorization[Authorization]
+for more information on access roles.
++
+Now, when the collaborator user signs in to
+link:https://manage.openshift.com[manage.openshift.com], they will see a card to
+log in to the web console for the same cluster as your subscription and, if the
+collaborator was granted access to your projects on the cluster, they will have
+access just like any other user.
+
+[[granting-project-access-using-the-cli]]
+=== Granting Project Access Using the CLI
+
+After xref:project-collaboration-adding-collaborators[adding the collaborator]
+to your {product-title} Pro subscription, you can grant project access to the
+collaborator using the CLI.
+
+. Log in to the cluster through the CLI using your access token.
+
+. Grant the user a role using the same user name listed on the *Collaboration* page using:
++
+----
+$ oc policy add-role-to-user <role-name> <username>
+----
++
+For example:
++
+----
+~$ oc login https://api.openshift.com --token=<...>
+Logged into "https://api.openshift.com:443" as "exampleuser" using the token provided.
+
+You have one project on this server: "exampleuser-collab"
+
+Using project "exampleuser-collab".
+~$ oc policy add-role-to-user view collaborator-1234
+role "view" added: "collaborator-1234"
+----
++
+This example grants *view* access to the project for user `collaborator-1234`.
+See
+xref:../architecture/additional_concepts/authorization.adoc#architecture-additional-concepts-authorization[Authorization]
+for more information on access roles.
++
+Now, when the collaborator user signs in to
+link:https://manage.openshift.com[manage.openshift.com], they will see a card to
+log in to the web console for the same cluster as your subscription and, if the
+collaborator was granted access to your projects on the cluster, they will have
+access just like any other user.
+
+[[project-collaboration-removing-collaborators]]
+=== Removing Collaborators
+
+If at any time you wish to remove the user as a collaborator from your
+subscription, you can do so on the same *Collaboration* page you used to add
+them. It is important to note, however, that this will not automatically remove
+any roles you have assigned the user in your projects. These will need to be
+manually deleted, or the user may still have access to your projects.
```

Comment:
Should we link to https://access.redhat.com/solutions/3094651 as docs on how to do this?

Reply:
Or is this KCS now deprecated by the next section[s] of docs?

Reply:
@sferich888 Doesn't CEE already have tier-2 access to be able to access all resources within a user's projects on all Pro clusters?